### PR TITLE
[JENKINS-39067] Move the GitHub icons to the github-branch-source plugin

### PR DESCRIPTION
**Pull request by stephenc**
See [JENKINS-39067](https://issues.jenkins-ci.org/browse/JENKINS-39067)

Upstream of https://github.com/jenkinsci/github-organization-folder-plugin/pull/32

@reviewbybees

### CODE REVIEW

**Comment by abayer** [comment]
:bee:

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgIcon.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+    }
+
+    @Extension(optional = true)
+    public static class DescriptorImpl extends FolderIconDescriptor {
+        @Override
+        public String getDisplayName() {
+            return Messages.GitHubOrgIcon_DisplayName();
+        }
+    }
+}
```
Comment (recena) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgIcon.java]: Reformatting?

**Comment by recena** [comment]
🐝

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgIcon.java (hunk 1 above)
Comment (stephenc) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubOrgIcon.java]: nope, moved from another repo

File: src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubLink.java [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import org.jenkins.ui.icon.IconSet;
+import org.jenkins.ui.icon.IconSpec;
+import org.kohsuke.stapler.Stapler;
+
+/**
+ * Link to GitHub
+ *
+ * @author Kohsuke Kawaguchi
+ */
+public class GitHubLink implements Action, IconSpec {
```
Comment (jglick) [inline src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubLink.java]: Best to remove `public` modifiers from classes like this.

**Pull request merged**